tests/NovelRT.Tests/CMakeLists.txt

```diff
@@ -38,4 +38,30 @@ target_link_libraries(Engine_Tests
     GMock::Main
 )
 
+add_custom_command(
+  TARGET Engine_Tests POST_BUILD
+  COMMAND ${CMAKE_COMMAND} -E copy_directory
+    $<TARGET_FILE_DIR:Dotnet>
+    $<TARGET_FILE_DIR:Engine_Tests>/dotnet
+)
+
+#this is pure hacky hotfix goodness. We need to figure out a better way to do this in the future.
```

Comment:
Does this comment only apply to the Nethost workaround still? Or are you referring to all of the Post Build steps now?

Reply:
This is purely the win32 nethost workaround.